website/docs/d/tenant_policies_ipsla_monitoring_policy.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "mso"
+page_title: "MSO: mso_tenant_policies_ipsla_monitoring_policy"
+sidebar_current: "docs-mso-data-source-tenant_policies_ipsla_monitoring_policy"
+description: |-
+Data source for IPSLA Monitoring Policy.
```

Comment:
should we mention tenant policy location in the description, comment also applies to resource in that case.

Reply:
Shall we introduce a new header GUI Info just like ACI for location?

Reply:
that is a good idea I think